frontend/src/libs/components/button/button.tsx

```diff
@@ -48,6 +52,11 @@ const Button: React.FC<Properties> = ({
     small: styles.buttonSmall,
   };
 
+  const outlineColorClassNameMapper: Record<ButtonOutlineColor, string> = {
+    green: styles.greenOutline,
+    white: styles.whiteOutline,
```

Comment:
Could it be styles.green\white? as it's not only about border, but also text color